_posts/2017-09-05-DevSwap.markdown

```diff
@@ -0,0 +1,109 @@
+
+Earlier this summer, the engineer leads received some feedback from our engineers about wanting more cross-team collaboration and knowledge sharing. In our weekly engineering-wide meeting, we get to hear what features each team is working on, but it can sometimes feels like there's missing context. Within our practices, there is a better sense of the details, but usually the work itself is deeply nested in another team.
+
+We brainstormed several ideas related to knowledge sharing and cross-team collaboration. What stood out the most was that being in product teams meant less opportunities to work in other parts of the business. This led to the question: What would happen if we redistributed ourselves into entirely new teams? Today I'll introduce something new we tried, something we're calling DevSwap.
+
+<!-- more -->
+
+## Setup
+
+Over the course of a few weeks, the engineer leads devised a plan for DevSwap. Here are some questions we asked ourselves:
+
+* How can we become closer as a team while learning something new?
+* How can we keep ourselves as engineers nimble, flexible, and full-stack?
+* How can we reduce risk in our systems by spreading knowledge?
+* How can we have fun before summer is over?
+
+For the first iteration, we decided to spend two full business days swapping. Every engineer  worked with a different team. Each team contained the same number of people as the original team. The leads of each team stayed to onboard, provide support and facilitate work.
+
+The Monday before DevSwap, we had an engineering-wide meeting to go over some guidelines and pick teams. Here are the guidelines we shared:
+
+* DevSwap will last 2 days. 🕑
+* Product managers have been notified so don't worry about your regular work. Hopefully a new member will pick some up for you 😄.
+* If possible, try and sit by your new team and if not, coordinate meeting times with remote members. 📞
+* See this as a pairing opportunity! 👫
+* In terms of work, it's business as usual. Your lead will give you guidance. 💼
+
+In this meeting, we also had our leads "pitch" their teams. The goal of this was to get people excited about joining new teams and to get a sense of what projects they might work on. To choose the actual teams, each engineer picked their top three choices and we drew names out of a bucket. If their first choice was already full, we would place them in their next available choice. The following Monday and Tuesday, we were all writing code for different teams.
+
+
+## So what happened?
+
+Our EU engineers kicked off DevSwap on Monday, tackling both the struggle of learning new stacks and being remote. Leads mitigated the latter problem by prepping EU folks on Friday so they had some direction come Monday morning.
```

Comment:
We should specify that we're based out of NYC, to give context for why we had to prep on Friday.